modules/ldap-auto-syncing.adoc

```diff
@@ -0,0 +1,246 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-auto-syncing_{context}"]
+= Automatically syncing LDAP groups
+
+You can automatically sync LDAP groups on a periodic basis by configuring a cron job.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured an LDAP identity provider (IDP), which involved creating an LDAP secret and config map.
+
+// TODO Any other prereqs? what about anything else w/ the sync stuff, like a config file, etc? Looks like they embed it in the config map
+
+.Procedure
+
+. Create a namespace where the cron job will run:
++
+[source,terminal]
+----
+$ oc create namespace ldap-sync <1>
+----
+<1> This procedure uses a namespace called `ldap-sync`.
+
+. Copy the secret and config map that you created when configuring the LDAP identity provider to this new namespace.
++
+The secret and config map exist in the `openshift-config` namespace and must be copied to the new `ldap-sync` namespace.
++
+// TODO: Provide actual instructions on how to copy, or just leave it like this?
```

Comment:
@stlaz At first I was trying to find an easy way/command that we could show on how to copy these over to the new namespace. But it involved getting the YAML and creating from that YAML, but then I gave up because you'd have to strip out some of the fields and change the namespace. So let me know your thoughts on if I should just leave this here as is, generically saying to copy it, or if you have any suggestions for an easy way they can copy it.

Reply:
something like https://gist.github.com/simonswine/6bf3b665e4117f42b550c3ea12dd171a might be possible but I'd probably just leave it as is